crates/router/src/connector/worldline/transformers.rs

```diff
@@ -130,12 +112,30 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for PaymentsRequest {
     }
 }
 
+#[derive(Clone, Debug, Eq, Hash, PartialEq, Deserialize, Serialize)]
+pub enum Gateway {
+    Amex = 2,
+    Discover = 128,
+    MasterCard = 3,
+    Visa = 1,
+}
+
+impl From<utils::CardIssuer> for Gateway {
+    fn from(issuer: utils::CardIssuer) -> Self {
+        match issuer {
+            utils::CardIssuer::AmericanExpress => Self::Amex,
+            utils::CardIssuer::Master => Self::MasterCard,
+            utils::CardIssuer::Discover => Self::Discover,
+            _ => Self::Visa,
```

Comment:
Default should throw error saying PaymentMethod/Issuer not supported, as this can prevent Create Payment api call with wrong product ID.